### ISSUE TITLE
Incorrect Next track ID after import chapters

### ISSUE THREAD

**Issue by L2501** [sandreas/tone#51]
in movie header atom after tagging chapters to m4b file the Next track ID is incorrectly set to "2" but ID 2 is already in use.

mediainfo --Details=10 file.m4b | grep "k ID"
```
0000008C   Next track ID:                        2 (0x00000002)
000000AC    Track ID:                            1 (0x00000001)
008995BD    Track ID:                            2 (0x00000002)
```
Ref: https://developer.apple.com/library/archive/documentation/QuickTime/QTFF/QTFFChap2/qtff2.html#//apple_ref/doc/uid/TP40000939-CH204-32947

**Comment by sandreas** [comment]
Thanks for reporting. I have to show this to the atldotnet maintainer, this may be fixed in a recent version.

**Comment by sandreas** [comment]
@L2501 
Mmh I was not able to reproduce this. Could you help me out with a (smallish) file and the according command that produces this problem?

**Comment by L2501** [comment]
import from the mp4v2 format, any file should do
```tone tag file.m4b --meta-chapters-file file.chapters.txt```

**Comment by L2501** [comment]
@sandreas https://mega.nz/file/UiAVnKyK#A8AvSDr_k8u5LsBljLy4UmFcOCZ7-kh02LUTlLd3o-s

**Comment by sandreas** [comment]
@L2501 
Thanks, I'll take a look as soon as possible, but maybe we have to redirect this to atldotnet / @Zeugma440, I'll keep you up to date.

**Comment by sandreas** [comment]
Should be fixed in `0.2.0`.
